torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -894,6 +896,21 @@ Tensor* TensorExprKernel::computeValue(const torch::jit::Value* v) {
           });
     } break;
 
+    case aten::masked_fill: {
+      return computeThreeOperand(
```

Comment:
Just checking: does `masked_fill` obey standard broadcasting rules as implied by `computeThreeOperand`?  (Meaning -- I think -- that all of `input`, `mask`, and `value` are broadcasted to the same shape?

Reply:
They are all broadcasted to `input`. Since we have guarded based on the sizes,  we know that we have a successful execution and that `mask` and `value` are broadcastable to `input`. I suspect there are a number of ops for which the default `broadcastInputs` logic would need to be refined if we didnt know that we have a valid set of input shapes.